script/install_chromium.sh

```diff
@@ -10,24 +10,35 @@ readonly TARGET_DIR=$1
 # The build of Chromium used to test web functionality.
 #
 # Chromium builds can be located here: https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/
-readonly CHROMIUM_BUILD=768968
-# The ChromeDriver version corresponding to the build above. See
-# https://chromedriver.chromium.org/downloads
-# for versions mappings when updating Chromium.
-readonly CHROME_DRIVER_VERSION=84.0.4147.30
+#
+# Check: https://github.com/flutter/engine/blob/master/lib/web_ui/dev/browser_lock.yaml
+readonly CHROMIUM_BUILD=929514
+
+# The correct ChromeDriver is distributed alongside the chromium build above, as
+# `chromedriver_linux64.zip`, so no need to hardcode any extra info about it.
+readonly DOWNLOAD_ROOT="https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F${CHROMIUM_BUILD}%2F"
 
 # Install Chromium.
-mkdir "$TARGET_DIR"
-wget --no-verbose "https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F${CHROMIUM_BUILD}%2Fchrome-linux.zip?alt=media" -O "$TARGET_DIR"/chromium.zip
-unzip "$TARGET_DIR"/chromium.zip -d "$TARGET_DIR"/
+mkdir $TARGET_DIR
```

Comment:
Why did you remove the quotes here?

Reply:
Because I had no idea what I was doing, but I've done some reading now :P

Reply:
https://tldp.org/LDP/abs/html/quotingvar.html